roles/dev/tasks/main.yml

```diff
@@ -0,0 +1,107 @@
+---
+# Devlopment specific tasks
+
+- name: Transport Layer Security (setup_ssl)
+  include_tasks: setup_ssl.yml
+  when: setup_ssl
+
+# Maildev
+- name: Install PHP PEAR modules
+  become: true
+  pear: name={{item}} state=present
+  with_items: { Auth_SASL, mail, mail_mime, Net_SMTP }
+
+- name: Install Maildev for catching emails while developing
+  become: true
+  npm: name=maildev version=1.0.0-rc3 state=present global=yes
+  notify: restart maildev
+
+- name: Setup Maildev to start on reboot
+  become: true
+  file: state=link src={{ dir.script }}/init_maildev.sh path=/etc/init.d/maildev mode='755'
+
+- name: Create Maildev startup symlink
+  # TODO insserv Script maildev is broken incomplete LSB comment.
+  become: true
+  file: state=link src=/etc/init.d/maildev path=/etc/rc3.d/S99maildev
+
+# MariaDB
+- name: Save db credentials locally
+  # See https://dev.mysql.com/doc/refman/5.7/en/option-files.html
+  # TODO use vault instead
+  copy:
+    dest: ~/.my.cnf
+    content: |
+      [client]
+      user={{db.username}}
+      password={{db.password}}
```

Comment:
What process is using this file?

Reply:
if it's there, all mysql commands, but I'll drop it in favor of vault (encrypted variable storage). question then is how to share the password between developers in the future.

Reply:
We don't have too many situations when we'd have to enter SQL credentials (only when provisioning the production, really). For the dev env these can be just something what we can push to git, like root/root.

Reply:
I.e. keeping it simple might be better. ;-)